CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [1.6.2] - 2022-10-20
```

Comment:
Was v1.6.2 actually deployed anywhere? If not, this should stay under "Unreleased". If so, in general we should try to deploy only off of the `develop` branch to avoid confusion about what we are running against when using the dev environment.

Reply:
v1.6.2 is not deployed anywhere. I just used the moment to increment the version once it's merged to develop

Reply:
Removed